src/components/typeform/typeform.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { CloseIcon } from "../icons";
+
+import * as styles from "./typeform.module.scss";
+
+const TypeForm = ({ onClick }) => {
+  return (
+    <div className={styles.formContainer}>
+      <button onClick={onClick} className={styles.closeForm}><CloseIcon className={styles.closeFormIcon} /></button>
+      <iframe
+        src="https://form.typeform.com/to/c7G2RUWm?typeform-embed=popup-blank&typeform-source=blog.xmartlabs.com&typeform-medium=embed-sdk&embed-hide-footer=true&typeform-embed-id=rclfl" 
```

Comment:
maybe you can add this as a constant or in env file

Reply:
Done